pandas/tests/dtypes/cast/test_upcast.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+from pandas.core.dtypes.cast import maybe_upcast_putmask
+
+from pandas import Series
+from pandas.util import testing as tm
+
+
+def test_upcast_series():
```

Comment:
can you parameterize these (both tests) as I suspect a myriad of cases (just a couple for now, e.g. int -> float you can start with)